pages/1.13/security/oss/login/service-login/index.md

```diff
@@ -0,0 +1,95 @@
+
+The secure service login is enabled through a service login token being signed by the private key of the service that generated it.
+
+# Login using the DC/OS CLI 
+
+**NOTE**: This demonstrates manual login for service account testing. A service login token is generated internally for the login process.
+
+**Prerequisite:**
+- [DC/OS CLI](/1.13/cli/)
+
+Using the [DC/OS CLI](/1.13/cli/) one can log in as service by specifying the `dcos-services` login provider.
+
+To test a service login using the DC/OS CLI specify the `dcos-services` login provider with the [auth login](/1.13/cli/command-reference/dcos-auth/dcos-auth-login/) command. Replace `<service-account-id>` and `<private-key-path>` with the corresponding values before executing the following command:
+
+```bash
+dcos auth login --provider=dcos-users --username=<service-account-id> --private-key=<private-key-path>
+```
+
+# Login using the IAM API
+
+## Construct a service login token
+
+For DC/OS, service login tokens must be RFC 7519 JSON Web Tokens (JWT) of type RS256.
+
+A service login token for DC/OS is constructed by adding the service account ID via the (`uid`) claim and the expiration time in Unix seconds via the (`exp`) claim to the JWT payload. The JWT header must specify the RS256 algorithm. The token is signed with the service account's private key.
+
+1. Header
+```json
+{
+    "alg": "RS256",
+    "typ": "JWT"
+}
+```
+
+2. Payload
+```json
+{
+    "uid": "<uid>",
+    "exp": <expiration_time>
+}
+```
+
+Service login tokens can be created manually via [jwt.io](https://jwt.io) or via your favorite JWT library.
+
+**NOTE**: Service login tokens and DC/OS Authentication tokens are both JWTs of type RS256 with identical claims. However they are not interchangable. A service login token is signed by a service private key while a DC/OS Authentication token is signed by the IAM private key.</p>
+
+## Login using a service login token
+
+**Prerequisite:**
+- [Service login token](/1.13/security/oss/authentication/authentication-token/service-login/#generate-a-service-login-token)
+
+During a service login a service login token is sent to the DC/OS [Identity and Access Management (IAM) API](/1.13/security/oss/iam-api/).
+
+**NOTE**: The IAM will reject service login tokens with a life-time greater than ten minutes.
+
+1. To log in a service supply `<service-account-id>` and `<service-login-token>` before executing the following command:
+
+```bash
+curl -k -X POST https://<host-ip>/acs/api/v1/auth/login -d '{"uid": "<service-account-id>", "token": "<service-login-token>"}' -H 'Content-Type: application/json'
+```
+
+2. A DC/OS Authentication token will be returned in the HTTP response body similar to the one below:
```

Comment:
Rewrite` A DC/OS Authentication token will be returned in the HTTP response body similar to the one below:`
with `A DC/OS Authentication token will be returned in the HTTP response body similar to the below example.`